src/components/box/box.pw.tsx

```diff
@@ -0,0 +1,897 @@
+  });
+
+  sizeConstants.forEach(([pixels]) => {
+    test(`should verify height and width are ${pixels} when size prop is passed as a string`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<Default size={pixels} />);
+      const boxElement = await getDataElementByValue(page, "box");
+      await assertCssValueIsApproximately(
+        boxElement,
+        "width",
+        parseInt(pixels)
+      );
+      await assertCssValueIsApproximately(
+        boxElement,
+        "height",
+        parseInt(pixels)
+      );
+    });
+  });
+
+  ["block", "inline-block", "flex", "contents", "list-item", "none"].forEach(
+    (display) => {
+      test(`should verify display is ${display}`, async ({ mount, page }) => {
+        await mount(<Default display={display} />);
+        const boxElement = await getDataElementByValue(page, "box");
+        await expect(boxElement).toHaveAttribute("display", display);
+        await expect(boxElement).toHaveCSS("display", display);
+      });
+    }
+  );
+
+  [
+    "baseline",
+    "bottom",
+    "middle",
+    "sub",
+    "super",
+    "text-bottom",
+    "text-top",
+    "top",
+  ].forEach((alignment) => {
+    test(`should verify verticalAlign is ${alignment}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<BoxComponentMulti verticalAlign={alignment} />);
+      const boxElement = await getDataElementByValue(page, "box");
+      await expect(boxElement).toHaveCSS("vertical-align", alignment);
+    });
+  });
+
+  ["auto", "clip", "hidden", "scroll", "visible"].forEach((overflow) => {
+    test(`should verify overflow is ${overflow}`, async ({ mount, page }) => {
+      await mount(<Default overflow={overflow} />);
+      const boxElement = await getDataElementByValue(page, "box");
+      await expect(boxElement).toHaveAttribute("overflow", overflow);
+      await expect(boxElement).toHaveCSS("overflow", overflow);
+    });
+  });
+
+  (["auto", "clip", "hidden", "scroll", "visible"] as const).forEach(
+    (overflow) => {
+      test(`should verify overflowX is ${overflow}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<Default overflowX={overflow} />);
+        const boxElement = await getDataElementByValue(page, "box");
+        await expect(boxElement).toHaveCSS("overflow-x", overflow);
+      });
+    }
+  );
+
+  (["auto", "clip", "hidden", "scroll", "visible"] as const).forEach(
+    (overflow) => {
+      test(`should verify overflowY is ${overflow}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<Default overflowY={overflow} />);
+        const boxElement = await getDataElementByValue(page, "box");
+        await expect(boxElement).toHaveCSS("overflow-y", overflow);
+      });
+    }
+  );
+
+  [
+    [200, 300],
+    [400, 400],
+  ].forEach(([width, minWidth]) => {
+    test(`should verify when number Width is set to ${width} that width is not less than minWidth ${minWidth}`, async ({
```

Comment:
**nitpick**: What do you think about amending the test description slightly, so its a bit clearer what the preconditions are and what is being asserted? 

```suggestion
    test(`when minWidth prop is 300 and width prop is ${width}, the calculated width should be greater than ${minWidth}px`, async ({
```

Reply:
I'll stick with "not less than" rather than "greater" (which isn't strictly accurate) - but otherwise I like your suggested rewriting 👍 I'll amend the other descriptions which are like this too.

Reply:
no, in fact it's an equality assertion in both cases (albeit an approximate one), so I'll remove any "greater/less than" in the description